app/frontend/src/__mocks__/service.ts

```diff
@@ -0,0 +1,42 @@
+import { service as originalService } from "../service";
+
+export const mockedService = {} as typeof originalService;
+
+export const service = new Proxy(
+  {},
+  {
+    get(target: {}, name: PropertyKey): any {
+      return new Proxy(
+        {},
+        {
+          get(target: {}, serviceName: PropertyKey): any {
+
+            const serviceMethod =
+              (mockedService as any)[name] &&
+              (mockedService as any)[name][serviceName];
+            if (serviceMethod) {
+              return async(...args:any[]) => {
+                  console.log(
+                      `Service method '${String(name)}.${String(
+                          serviceName
+                      )}' is called with args:`,...args
+                  );
+                  const result=await serviceMethod(...args)
+                  console.log('Result: ', result)
+                  return Promise.resolve(result);
+              };
+            } else
```

Comment:
In what instances is `serviceMethod` false? Also did prettier do this formatting? I find `else` with no `{` rather odd.

Reply:
It is false by default as can be seen in the second line in this file. The developer might need to define it.

Reply:
I fixed the {